style.css

```diff
@@ -0,0 +1,117 @@
+:root {
+    --black: #222426;
+    --gray-1: #525960;
+    --gray-2: #6a737c;
+    --gray-3: #9fa6ad;
+    --gray-4: #d7d9dc;
+    --gray-5: #e4e6e8;
+    --gray-6: #f1f2f3;
+    --white: #ffffff;
+    --white-gray: #f8f9f9;
+    --navy: #39739d;
+    --dark-blue: #0074cc;
+    --blue: #0a95ff;
+    --lightblue: #e1ecf4;
+    --primary-orange: #f48225;
+}
+
+/* nav bar up */
+
+body {
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+    margin: 0;
+}
+
+.search-icon {
+    font-size: inherit;
+    position: absolute;
+    left: 1em;
+    bottom: 0.7em;
+    width: 1em;
+    fill: var(--gray-2);
+}
+
+form {
+    position: relative;
+}
+
+nav {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-wrap: wrap;
+    background-color: var(--white-gray);
+    box-shadow: rgba(0, 0, 0, 0.2) 0 0 5px 0px;
+    border-top: var(--primary-orange) solid 3px;
+    font-size: 15px;
+    column-gap: 0.7em;
+
+}
+
+nav a {
+    text-decoration: none;
+    color: var(--gray-2);
+
+}
+
+nav p {
+    color: var(--gray-2);
+}
```

Comment:
I think we can improve this a bit. Le's add the `color: var(--gray-2` rule on the `nav`, and delete those 2 rules.

Then, add a new rule regarding all `<a>` tags, and have them inherit the color, and also remove text decoraiton

```
a {
    text-decoration: none;
    color: var(--gray-2);
}
```

We should have the same result, but with less code.